### PR TITLE
added admin/edit route

### LINKED ISSUE #42: Edit problems to the database
Route: http://localhost:3000/admin/edit/:qID
The app is connected to the database: `./config/database.js`. Edit the problems to the database. The schema is in `./models/problems.js` and `./models/testcases.js`.
Basically, you'll have to implement the function: `editQuestion()` in the file `./controls/admin.js`. You can add other helping functions there.

### PR DESCRIPTION
**Pull request by Milind712000**
# What's this PR do? (related issue if exists)
updated admin/edit route
resolve #42 
# Any context you want to provide on the implementation?

# Screenshots? (necessary if an UI/UX change)

### CODE REVIEW

File: routes/admin.js [hunk 1]
```diff
@@ -10,8 +12,10 @@ router.get("/add", (req, res, next) => {
     res.render("problem_add");
 });
 
-router.get("/edit/:qID", (req, res, next) => {
-    res.render("problem_edit");
+router.get("/edit/:qID",async (req, res, next) => {
+    const ques = await Question.findOne({"qID":req.params.qID});
+    const t_case = await TC.findOne({"qID":req.params.qID});
+    res.render("problem_edit",{ques,t_case});
 });
```
Comment (Vishal1541) [inline routes/admin.js]: Put this in a function `editQuestion` which is in `./controls/admin.js`

Comment (Milind712000) [reply routes/admin.js]: these have to be passed to the page before rendering, so I'd have to create a different route just to pass the problem statement and testcase.
It is more readable this way

Comment (Vishal1541) [reply routes/admin.js]: Oh! My bad. I thought it was `router.put`. You can then put this in a function `helper.getQuestion = async (req, res, next) =>{...}` and then the `admin.js` will look clean: `router.get("/edit/:qID", admin.getQuestion);`

File: routes/admin.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 var express = require("express");
 var router = express.Router();
 var admin = require("../controls/admin");
+const Question = require("../models/problems");
+const TC = require("../models/testcases");
```
Comment (Vishal1541) [inline routes/admin.js]: Also, then you don't need to require problems and testcases (Line 4 and 5).

Comment (Milind712000) [reply routes/admin.js]: Then I won't be able to fetch the question and testcase from the database

Comment (Vishal1541) [reply routes/admin.js]: Why not. It's working perfectly fine.
a
b

Comment (Milind712000) [reply routes/admin.js]: Okay , previously I saw your comment to shift to editQuestion(put) so at that time it was required,
You're right,
If I also shift the function to handle the request getQuestion then It won't be required,
I'll do it

**Pull request merged**